_pytest/test_featurizers.py

```diff
@@ -59,3 +60,33 @@ def test_ngram_featurizer(spacy_nlp):
                            max_number_of_ngrams=10)
     assert len(ftr.all_ngrams) > 0
     assert ftr.best_num_ngrams > 0
+
+
+@pytest.mark.parametrize("sentence, expected, labeled_tokens", [
+    ("hey how are you today", [0., 1.], [0]),
+    ("hey 123 how are you", [1., 1.], [0, 1]),
+    ("blah balh random eh", [0., 0.], []),
+    ("looks really like 123 today", [1., 0.], [3]),
```

Comment:
so what do we give to the CRF in terms of features? just a binary feature for each token which says whether *any* regex matched it?